src/javascript/app/pages/cashier/account_transfer.js

```diff
@@ -225,30 +223,52 @@ const AccountTransfer = (() => {
                         getElementById(messages.parent).setVisibility(1);
                         return;
                     }
-                    max_amount = Currency.getMaxTransfer(
-                        Client.get('currency')
+                    max_amount = Currency.getTransferLimits(
+                        Client.get('currency'),
+                        'max'
                     );
                     transferable_amount = max_amount ?
                         Math.min(max_amount, withdrawal_limit, client_balance) :
                         Math.min(withdrawal_limit, client_balance);
 
-                    getElementById('range_hint').textContent = `${localize('Min')}: ${min_amount} ${localize('Max')}: ${transferable_amount.toFixed(Currency.getDecimalPlaces(Client.get('currency')))}`;
-
-                    el_explain_dynamic_limits.innerHTML = localize(
-                        'Maximum transferable amount will be chosen from a minimum value of:[_4] Current balance: [_1][_4]Daily withdrawal limit: [_2][_4]Current account\'s maximum allowed amount: [_3]]',
-                        [
-                            Currency.formatMoney(Client.get('currency'), client_balance),
-                            Currency.formatMoney(Client.get('currency'), withdrawal_limit),
-                            max_amount ? Currency.formatMoney(Client.get('currency'), max_amount) : localize('Not announced for this currency.'),
-                            '<br />',
-                        ],
+                    getElementById('range_hint_min').textContent = min_amount;
+                    getElementById('range_hint_max').textContent = transferable_amount.toFixed(
+                        Currency.getDecimalPlaces(
+                            Client.get('currency')
+                        )
                     );
+                    populateHints();
                     populateAccounts(accounts);
                 });
             }
         });
     };
 
+    const populateHints = () => {
+        getElementById('limit_current_balance').innerHTML  = Currency.formatMoney(
```

Comment:
`textContent` isn't enough? we need `innerHTML`?

Reply:
formatMoney produces HTML, so it'll get escaped.